lib/jest-helpers.js

```diff
@@ -12,12 +12,24 @@ nunjucks.configure(configPaths.components, {
   lstripBlocks: true
 })
 
+/**
+ * Render a component's template for testing
+ * @param {string} componentName
+ * @param {string} params parameters that are used in the component template
+ * @returns {object} returns object that includes raw HTML (output) and
+ * also a cheerio (jQuery) instance of the template for easy DOM querying
+ */
 function render (componentName, params) {
   const output = nunjucks.render(componentName + '/template.njk', { params })
```

Comment:
Wonder if we need this output anymore since it was originally used for snapshot testing...